### PR TITLE
Added site collection quota information to private-channels.md

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4575

I have added this sentence: "These site collections are created with a 25TB storage quota and cannot be changed."

@LanaChin could you help to confirm this information? I verified that they are created with a storage quota of 25TB but I´m not 100% sure that it cannot be changed. I tried with Set-SPOSite, no error but the storage quota didn´t change.

### CODE REVIEW

**Comment by LanaChin** [comment]
Followed up with PM

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/private-channels.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ See [Manage the life cycle of private channels in Teams](private-channels-life-c
...
+Each private channel has its own SharePoint site collection optimized for file sharing and fast provisioning. The separate site collection is to ensure access to private channel files is restricted to only members of the private channel compared to the team site where team owners have access to all the assets within the site collection. These site collections are created with a document library by default, and can be easily enhanced to a full-featured site collection through the [site management interface](https://support.office.com/article/Enable-or-disable-site-collection-features-A2F2A5C2-093D-4897-8B7F-37F86D83DF04). Each site collection is created in the same geographic region as the site collection of the parent team. These lightweight sites have a custom template ID, "TEAMCHANNEL#0", for easier management through PowerShell and Graph API. By design, they aren't visible in the SharePoint admin center. These site collections are created with a 25TB storage quota and cannot be changed.
```
Comment (JohanFreelancer9) [inline Teams/private-channels.md:132]:
```suggestion
Each private channel has its own SharePoint site collection optimized for file sharing and fast provisioning. The separate site collection is to ensure access to private channel files is restricted to only members of the private channel compared to the team site where team owners have access to all the assets within the site collection. These site collections are created with a document library by default, and can be easily enhanced to a full-featured site collection through the [site management interface](https://support.office.com/article/Enable-or-disable-site-collection-features-A2F2A5C2-093D-4897-8B7F-37F86D83DF04). Each site collection is created in the same geographic region as the site collection of the parent team. These lightweight sites have a custom template ID, "TEAMCHANNEL#0", for easier management through PowerShell and Graph API. By design, they aren't visible in the SharePoint admin center. These site collections are created with a 25 TB storage quota and cannot be changed.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dariomws** [comment]
Talked with Lana and we think it´s better to close this PR and not apply this change.

**Comment by rachoksh** [comment]
What was the reason behind not updating the documentation?

I think when tenant is set for Automatic quota management we get a 25 TB default quota applied and cannot be changed. 
When tenant is set for Manual quota I was able to change with PowerShell using below command.

Set-SPOSite -Identity https://abc123.sharepoint.com/sites/My_Cool_Team-TeamRoster -StorageQuota 20480 -StorageQuotaWarningLevel 17000

**Comment by dariomws** [comment]
Hi @rachoksh,

Yes, but it´s related to any SPO site collection, not only to Teams site collections. This is why we think it's redundant information to add it here.

**Comment by rachoksh** [comment]
Ok, got it! Thanks

**Comment by dariomws** [comment]
@rachoksh thanks again, for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Pull request closed without merging**